### DISCUSSION TITLE
How to show both "Table of contents"(on the left) and "On this page"(on the right)?

### DISCUSSION THREAD

**Discussion by hopezh** [quarto-dev/quarto-cli#5871] [Q&A] [answered]
### Description

May I ask how to create a doc that shows both `toc` (red box) and `On this page` (orange box), like the `fastai` web page shown below?

Thanks. 

![image](https://github.com/quarto-dev/quarto-cli/assets/23435810/6161c708-922b-42a7-9f39-e5c4e153226f)

**Comment by mcanouil** [comment, marked as answer]
I am not sure to understand.
This is the default behaviour in Quarto website.

Side navigation (see <https://quarto.org/docs/websites/website-navigation.html#side-navigation>) + ˋtoc: true`.

**Reply by hopezh** [reply]
Yes, indeed. It's generated by the website template using `quarto create-project mysite --type website`. Thanks. 

![image](https://github.com/quarto-dev/quarto-cli/assets/23435810/5490d185-6e85-425f-907b-de4a57cac51c)
